test/dotcom/system_status/subway_test.exs

```diff
@@ -513,68 +652,4 @@ defmodule Dotcom.SystemStatus.SubwayTest do
   defp between(time1, time2) do
     Faker.DateTime.between(time1, time2) |> Timex.to_datetime("America/New_York")
   end
-
-  # Returns a random alert that will be active at the time given by
-  # the required `:time` opt.
-  #
-  # Required opts:
-  #  - route_id
-  #  - time
-  #
-  # Optional opts:
-  #  - effect (default behavior is to choose an effect at random)
-  defp current_alert(opts) do
```

Comment:
non-blocking: I liked these functions you removed, as they made the tests themselves more concise 😅 Miss them.